### PR TITLE
Add unit testing configuration and two unit tests

### PR DESCRIPTION
**Pull request by jellebouwman**

### CODE REVIEW

File: components/timesheet-input/__tests__/TimesheetInput-test.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import TimesheetInput from '../TimesheetInput';
+
+test('ComponentName', () => {
+  const timesheetInput = shallow(<TimesheetInput prop="value" />);
```
Comment (RobbinHabermehl) [inline components/timesheet-input/__tests__/TimesheetInput-test.jsx]: `prop` doesn't seem to be a valid prop? It's important to create actual use-cases.

File: package.json [hunk 2]
```diff
@@ -7,7 +7,8 @@
     "build": "next build",
     "dev": "next & json-server --watch api/time-entries.json",
     "lint": "eslint --ext js,jsx .",
-    "start": "next start"
+    "start": "next start",
+    "test": "jest"
```
Comment (RobbinHabermehl) [inline package.json]: You seem to miss the Node environment. Does it work this way?

File: services/convert-time/__test__/convert-time.spec.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { convertDateToIso, convertTimeToIso } from '../convert-time';
+
+test('Split, reverse and join Dutch date notation', () => {
+  expect(convertDateToIso('01-01-2000')).toBe('2000-01-01');
```
Comment (boris-arkenaar) [inline services/convert-time/__test__/convert-time.spec.js]: Use different numbers for day and month, to make sure both are moved to the right place.

File: package.json (hunk 2 above)
Comment (jellebouwman) [reply package.json]: Was running properly, but I've added the `Node env` and added a `--watch`.

**Pull request merged**